ci/check-compiles.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -e
+set -x
```

Comment:
The current progress is hidden as the rebase progresses. I tested a few things locally and I think something like `git log -1 --oneline` at the top here is enough to let me know where we are in the stack.

I also really like the side-effect of being able to fix my issue in the middle of a rebase instead of having to create a temp branch, fix, rebase.

Reply:
Hmm, if it fails it should leave it where it was when it failed, no?

Reply:
Yes, when it fails it leaves it in the rebase where I can make changes which is great.

When it doesn't fail it is still nice to see the progress which explains the --oneline